README.md

```diff
@@ -1,105 +1,88 @@
+  2. Change into the directory and run `npm install` to install project dependencies.
+  3. Run `npm start` to start a local server 
+  4. To access the site in your browser, navigate to the localhost address that your terminal gives you. 
+  5. For any of the class tests, run ```npm test [name of the test you want to run]``` 
+  6. Enter `control + c` in your terminal to stop the server at any time.
 
-**Put all of your test files in the `test` directory.** As a convention, all test filenames should end with `-test.js`. For instance: `box-test.js`.
+## Contributors
 
-## Running Your Tests
+[This project](https://frontend.turing.io/projects/module-2/refactor-tractor-wc.html) was used exisiting code from previous students and staff at [Turing School Staff](https://turing.io/)
 
-Run your test suite using the command:
+This refactor was coded by [Jeff Kersting](https://github.com/JeffKersting), [Allison Dietz](https://github.com/dietza), [Thao Ma](https://github.com/thaomonster) and [Alice Ruppert](https://github.com/srslie).
 
-```bash
-npm test
-```
+Special thanks to our mentors [Scott Schipke](https://github.com/sschipke) 
 
-The test results will output to the terminal.
+//ADD MENTORS?
 
----
+## Technologies
 
-## Linting Your Code
+During this project we gained experience with the following technologies:
+- ES6 JavaScript
+- Testing with Mocha and Chai
+- Linting with ESLint
+- Semantic HTML
+- Responsive SCSS with Media Queries
+- Wave, ColorBlind, and ARIA Accessibility
+- Using network requests with API endpoints and async JavaScript
+- Using Webpack to compile mulitple class and script files
+- Deploying a site on GitHub Pages
+- Git workflow using GitHub Issues and Project Kanban Board
 
-Run the command in your terminal `npm run lint` to run the linter on your JavaScript code. There will be errors and warnings right from the start in this starter kit - the linter is still running successfully.
+We also gained invaluable experience working with a larger group (4 people) and collaborating on exisiting data.
+## Future Iterations
 
-Your linter will look at the JavaScript files you have within the `src` directory and the `test` directory. 
+This project helped us practice familiarizing ourselves with exisiting code, working asynchronously with three others a project in a relatively short time-frame, and honed our planning, communication and project managament.
 
-## Webpack?
+However, if time permitted, we would've loved to accomplish more. 
 
-If you look in the `package.json` file, you'll see one of the library dependencies called `webpack`. If you're interested in learning more about what Webpack is and how it works behind the scenes, take a look through the [Webpack configuration documentation](https://webpack.js.org/concepts/).
+While no future work is planned on this project, we kept in mind ideas for building this out for further functionality.
```

Comment:
and that we could probably lose this piece as we are actually listing suggestions for further development below

Reply:
Kept the idea that we're not doing future work but reworded.